src/app/modules/item/task/rxjschannel.ts

```diff
@@ -0,0 +1,75 @@
+import { build, ChannelConfiguration, MessageTransaction, MessagingChannel } from "jschannel";
```

Comment:
I would prefer you put this files in a more explicit directory, for instance  `src/app/modules/item/task-communication` (or a better name)

Reply:
Sure, I'll rename them. I'm not sure whether GitHub will transfer comments properly after a rename, so I'll do that once this PR is closer to completion.